crypto/sql_store.go

```diff
@@ -668,9 +668,9 @@ func (store *SQLCryptoStore) PutDevice(ctx context.Context, userID id.UserID, de
 // PutDevices stores the device identity information for the given user ID.
 func (store *SQLCryptoStore) PutDevices(ctx context.Context, userID id.UserID, devices map[id.DeviceID]*id.Device) error {
 	return store.DB.DoTxn(ctx, nil, func(ctx context.Context) error {
-		_, err := store.DB.Exec(ctx, "INSERT INTO crypto_tracked_user (user_id) VALUES ($1) ON CONFLICT (user_id) DO NOTHING", userID)
+		_, err := store.DB.Exec(ctx, "INSERT OR REPLACE INTO crypto_tracked_user (user_id, devices_outdated) VALUES ($1, FALSE)", userID)
```

Comment:
I think we need to maintain compatibility with postgres, and I don't think that this syntax works.

```suggestion
		_, err := store.DB.Exec(ctx, "INSERT INTO crypto_tracked_user (user_id, devices_outdated) VALUES ($1, FALSE) ON CONFLICT DO UPDATE SET devices_outdated=excluded.devices_outdated", userID)
```